great-idea-website/index.html

```diff
@@ -15,49 +15,89 @@
 </head>
 
   <body>
+    <div class="main-container">
+   
+      <nav>
+          <a href="services">Services</a>
+          <a href="product">Product</a>
+          <a href="vision">Vision</a>
+          <a href="features">Features</a>
+          <a href="about">About</a>
+          <a href="contact">Contact</a>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+      </nav>
+   
+      <header>
+        <div class="header-left">
+        <h1>Innovation
+          <br>
+            On
+          <br>
+            Demand
```

Comment:
I think using the <br/> tag makes it really easy to make it look like the design file.  good job using that.